diff --git a/app/app.go b/app/app.go
index 678ccc0a..ba64a7fe 100644
--- a/app/app.go
+++ b/app/app.go
@@ -1060,6 +1060,7 @@ func (app *App) setAnteHandler(txConfig client.TxConfig, maxGasWanted uint64, bl
 		},
 		ExtraDecorators:   []sdk.AnteDecorator{blockAddressDecorator},
 		PendingTxListener: app.onPendingTx,
+		UnsafeUnorderedTx: true,
 	}
 
 	anteHandler, err := evmante.NewAnteHandler(options)
