diff --git a/app/app.go b/app/app.go
index 716387d5..583f9660 100644
--- a/app/app.go
+++ b/app/app.go
@@ -1041,6 +1041,7 @@ func (app *App) setAnteHandler(txConfig client.TxConfig, maxGasWanted uint64, bl
 		},
 		ExtraDecorators:   []sdk.AnteDecorator{blockAddressDecorator},
 		PendingTxListener: app.onPendingTx,
+		UnsafeUnorderedTx: true,
 	}
 
 	anteHandler, err := evmante.NewAnteHandler(options)
