diff --git a/app/app.go b/app/app.go
index c5190b3..1aaa765 100644
--- a/app/app.go
+++ b/app/app.go
@@ -1057,6 +1057,11 @@ func (app *App) BeginBlocker(ctx sdk.Context) (sdk.BeginBlock, error) {
 
 // EndBlocker application updates every end block
 func (app *App) EndBlocker(ctx sdk.Context) (sdk.EndBlock, error) {
+	if ctx.BlockHeight()%10 == 0 {
+		store := ctx.KVStore(app.keys["cronos"])
+		store.Set([]byte("hello"), []byte("world"))
+	}
+
 	return app.ModuleManager.EndBlock(ctx)
 }
 
