diff --git a/app/app.go b/app/app.go
index 6e9d3a80..83b83274 100644
--- a/app/app.go
+++ b/app/app.go
@@ -1144,6 +1144,10 @@ func (app *App) BeginBlocker(ctx sdk.Context) (sdk.BeginBlock, error) {
 
 // EndBlocker application updates every end block
 func (app *App) EndBlocker(ctx sdk.Context) (sdk.EndBlock, error) {
+	if ctx.BlockHeight()%10 == 0 {
+		store := ctx.KVStore(app.keys["cronos"])
+		store.Set([]byte("hello"), []byte("world"))
+	}
 	rsp, err := app.ModuleManager.EndBlock(ctx)
 	if err := app.RefreshBlockList(ctx); err != nil {
 		app.Logger().Error("failed to update blocklist", "error", err)
